Jenkinsfile

```diff
@@ -2,12 +2,27 @@
 node {
   stage('checkout sources') {
         // You should change this to be the appropriate thing
-        git url: 'https://github.com/ColumbusStateWorkforceInnovation/special-topics-ci-lab'
+        //git url: 'https://github.com/ColumbusStateWorkforceInnovation/special-topics-ci-lab'
+        git url: 'https://github.com/esaboleyjr/specialtopics-ci-lab'
   }
 
   stage('Build') {
     // you should build this repo with a maven build step here
+    withMaven (maven: 'maven3') {
+      sh "mvn package"
+    }
     echo "hello"
   }
   // you should add a test report here
+
+  node {
+      try {
+          stage('Test') {
+              sh './gradlew check'
```

Comment:
Mine fails with:
```
[Pipeline] sh
+ ./gradlew check
/usr/share/jenkins/ref/jobs/build-pipeline-job/workspace@2@tmp/durable-142b538e/script.sh: 1: /usr/share/jenkins/ref/jobs/build-pipeline-job/workspace@2@tmp/durable-142b538e/script.sh: ./gradlew: not found
[Pipeline] }
[Pipeline] // stage
[Pipeline] archiveArtifacts
Archiving artifacts
‘build/libs/**/*.jar’ doesn’t match anything: even ‘build’ doesn’t exist
[
```

I sent out some updates earlier on this to the class - you should read through them.  You don't want to do the gradle check since this isn't built with gradle - it's built with maven.  Also the `build/` directory is a gradle-ism.  Maven puts its  build output in a different directory (again, this information can be found in an earlier announcement).